### PR TITLE
Remove dataplane dependency of oc commands

### PR DESCRIPTION
**Pull request by fao89**
ref: https://github.com/openstack-k8s-operators/dataplane-operator/pull/325

### CODE REVIEW

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

File: devsetup/scripts/gen-edpm-baremetal-kustomize.sh [hunk 1]
```diff
@@ -70,15 +70,12 @@ fi)
     - op: add
       path: /spec/roles/edpm-compute/nodeTemplate/ansibleSSHPrivateKeySecret
       value: dataplane-ansible-ssh-private-key-secret
-    - op: replace
+    - op: remove
```
Comment (slagle) [inline devsetup/scripts/gen-edpm-baremetal-kustomize.sh:73]: We could just remove these 4 vars from the samples in dataplane-operator as well. Then we wouldn't need to remove them here.

**Review by slagle — APPROVED** [review]

File: devsetup/scripts/gen-edpm-baremetal-kustomize.sh (hunk 1 above)
Comment (fao89) [reply devsetup/scripts/gen-edpm-baremetal-kustomize.sh:73]: I'll start a PR then

Comment (fao89) [reply devsetup/scripts/gen-edpm-baremetal-kustomize.sh:73]: https://github.com/openstack-k8s-operators/dataplane-operator/pull/331

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Review by slagle — COMMENTED** [review]
This LGTM, but should it wait for https://github.com/openstack-k8s-operators/dataplane-operator/pull/331?

**Comment by fao89** [comment]
actually it should wait for https://github.com/openstack-k8s-operators/openstack-operator/pull/426

**Comment by abays** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
Thank you @abays

**Comment by fao89** [comment]
recheck

**Comment by abays** [comment]
This has now passed all Zuul jobs independently across many attempts.

/override rdoproject.org/github-check

**Review by abays — APPROVED** [review]
/lgtm

**Comment by rabi** [comment]
> recheck
> 
> It looks like the `install-yamls-crc-podified-edpm-baremetal` job has never succeeded, otherwise I would override at this point. This may be due to the problem I noted here though [1].
> 
> [1] https://github.com/openstack-k8s-operators/install_yamls/pull/460/files#r1287423440

I think most of recent failures are due to leader election issues across different operator controllers other than failures requiring reauth.

[1] https://logserver.rdoproject.org/63/63/b2c5b6f814fe4edbf820724bc074f7e8ea8903ac/github-check/openstack-baremetal-operator-crc-podified-edpm-baremetal/cc98e18/ci-framework-data/logs/crc/pods/openstack-operators_nova-operator-controller-manager-5679b65dc4-lfzgj_88aaa028-7812-4120-9271-f2a897d2bca4/manager/11.log

**Pull request merged**